operator/controllers/csiscaleoperator_controller.go

```diff
@@ -2410,10 +2435,15 @@ func setDefaultDriverEnvValues(envMap map[string]string) {
 
 // getDiscoverCGFilesetDefaultValue returns default value for CG fileset discovery
 func getDiscoverCGFilesetDefaultValue() string {
-	_, isOpenShift := os.LookupEnv(config.ENVIsOpenShift)
-	if isOpenShift {
-		//CG fileset discovery is enabled by default on OpenShift cluster
-		return "ENABLED"
+	logger := csiLog.WithName("getDiscoverCGFilesetDefaultValue")
+	logger.Info("Getting DISCOVER_CG_FILESET default value")
+
+	if CNSAClusterPresence, ok := clusterTypeData[config.ENVClusterCNSAPresenceCheck]; ok {
```

Comment:
I feel significance of these 2 env variable (ENVClusterCNSAPresenceCheck and EnvDiscoverCGFilesetKey) is same, so should we use only one at now say just ENVClusterCNSAPresenceCheck

Reply:
It's better to keep both because EnvDiscoverCGFilesetKey can be used to disable CG fileset discovery on a CNSA setup for non-RDR usecases